src/ice_model.F90

```diff
@@ -2398,16 +2399,17 @@ subroutine ice_model_init(Ice, Time_Init, Time, Time_step_fast, Time_step_slow,
       ! When SPECIFIED_ICE=True, the variable Ice%sCS%OSS%SST_C is used for the skin temperature
       ! and needs to be updated for each run segment, regardless of whether a restart file is used.
       call get_sea_surface(Ice%sCS%Time, sG%HI, SST=Ice%sCS%OSS%SST_C, ice_domain=Ice%slow_domain_NH)
-
+      if (.not.is_restart) then
```

Comment:
The following block should be un-commented for specified-ice when starting cold (no restart file). Otherwise there will be a restart issue.